src/event.js

```diff
@@ -610,8 +610,7 @@ jQuery.event = {
 			event,
 			{
 				type: type,
-				isSimulated: true,
-				originalEvent: {}
```

Comment:
I think if it's not a native event, or even represents a native event, it
probably shouldn't be there, or we get into how we're going to simulate
native events. Consistency is not a strong enough argument to do that.
On Tue, May 19, 2015 at 09:47 Dave Methvin <notifications@github.com> wrote:

> In src/event.js
> <https://github.com/jquery/jquery/pull/2303#discussion_r30599937>:
>
> > @@ -610,8 +610,7 @@ jQuery.event = {
> >  			event,
> >  			{
> >  				type: type,
> > -				isSimulated: true,
> > -				originalEvent: {}
>
> I think that brings us back to the problem we were trying to avoid, which
> is that the "donor" event gets affected. That is why we wanted either a
> different empty event or no originalEvent.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/jquery/jquery/pull/2303/files#r30599937>.
>